packages/ai-commands/src/sql/cron.ts

```diff
@@ -0,0 +1,96 @@
+import type OpenAI from 'openai'
+import { ContextLengthError } from '../errors'
+import { jsonrepair } from 'jsonrepair'
+
+// Responds to a natural language request for cron syntax.
+export async function chatCron(openai: OpenAI, prompt: string) {
+  const initMessages: OpenAI.Chat.Completions.ChatCompletionMessageParam[] = [
+    {
+      role: 'system',
+      content: `
```

Comment:
Maybe we prefix with `codeBlock` template tag from `common-tags` which will strip indents (so that we're not passing indented prompts to OpenAI).